src/services/validations/validate.ts

```diff
@@ -189,13 +197,15 @@ const validateSendTransferAmount = (
     if (Number(amount) > 0 && selectedAsset.decimals && selectedAsset.decimals > 0) {
       return {
         success: false,
-        message: `The amount must be greater than 0.${'0'.repeat(selectedAsset.decimals - 1)}1.`
+        message: `The amount must be greater than 0.${'0'.repeat(selectedAsset.decimals - 1)}1.`,
+        severity: 'warning'
       }
     }
 
     return {
       success: false,
-      message: 'The amount must be greater than 0.'
+      message: 'The amount must be greater than 0.',
+      severity: 'warning'
```

Comment:
Why are all of these amount errors warnings? How do we determine which one is a warning, and which is an error